components/Layout/Nav/Nav.tsx

```diff
@@ -0,0 +1,30 @@
+import Link from 'next/link';
+import { List, ListItem, NavButton, NavIcon, NavTrigger } from './Nav.styles';
+
+export const Nav = () => {
+  return (
+    <nav role="navigation">
+      <NavButton className="nav-btn" type="checkbox" id="nav-btn" />
```

Comment:
Q: Why use `className`s here?

Reply:
Ah, looks like I missed removing this when cleaning up. Good catch!